dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc

```diff
@@ -315,7 +315,7 @@ void VerilatorSimCtrl::Run() {
                 << std::endl;
       break;
     }
-    if (term_after_cycles_ && time_ > term_after_cycles_) {
+    if (term_after_cycles_ && time_ / 2 >= term_after_cycles_) {
```

Comment:
nit: Personally I like to some parenthesis in nested expressions to make intent more obvious rather than relying on operator precedence alone, e.g.

```c
if (term_after_cycles_ && (time_ / 2 >= term_after_cycles_)) {
````

However I can't immediately find any guidance on this in the C++ style guide, so up to you.

Reply:
Thanks. I added the parenthesis.